source/Masonry/Masonry.js

```diff
@@ -328,11 +332,11 @@ export default class Masonry extends React.PureComponent<Props> {
     );
   }
 
-  _debounceResetIsScrollingCallback() {
+  _debounceResetIsScrollingCallback = () => {
```

Comment:
nit: some functions are arrows and some are not.

Reply:
It's actually intentional. I think I might have a lint rule to protect against unnecessary arrow functions. Not sure though. But I added arrow functions where I wanted to ensure the correct "this" even after passing references around

Reply:
Oh I see!